cnquery.go

```diff
@@ -42,6 +47,47 @@ func GetVersion() string {
 	return Version
 }
 
+// Release represents a GitHub release
+type Release struct {
+	Name string `json:"name"`
+}
+
+// GetLatestReleaseName fetches the name of the latest release from the specified GitHub repository
+func GetLatestReleaseName() (string, error) {
+	url := "https://api.github.com/repos/mondoohq/cnquery/releases/latest"
```

Comment:
nit: can we make this a param and rename to `defaultUrl`. we can reuse this func for cnspec as well, if needed